mypaint-brush.c

```diff
@@ -361,24 +361,25 @@ mypaint_brush_set_state(MyPaintBrush *self, MyPaintBrushState i, float value)
     self->states[i] = value;
 }
 
+// C fmodf function doesn't handle negative numbers properly
+float mod(float a, float N)
```

Comment:
Please make this function static and inline, to make it clearer that it is limited to just this translation unit and isn't intended for export. At your discretion, a clearer name or comment indicating how this differs from fmodf() would be helpful too.

Also, please remove the commented-out printf() line or make it active but wrapped in `#ifdef HEAVY_DEBUG`.